packages/modules/create-imodel/src/components/context/imodel-context.tsx

```diff
@@ -23,6 +23,16 @@ type ContextProps = {
   nameTooLong: string;
   descriptionString: string;
   descriptionTooLong: string;
+  /** Confirm button text */
+  confirmButtonText?: string;
+  /** Cancel button text */
+  cancelButtonText?: string;
+  /** Confirm action callback function */
+  confirmAction: () => void;
```

Comment:
why mandatory?

Reply:
because we are using that action from ButtonBar component, so it mandatory